sentry/src/main/java/io/sentry/ISentryClient.java

```diff
@@ -266,5 +266,6 @@ SentryId captureTransaction(
   }
 
   @NotNull
+  @ApiStatus.Experimental
```

Comment:
I think the `@ApiStatus.Experimental` Annotation is missing on the `SentryClient.java` class